pom.xml

```diff
@@ -155,6 +159,27 @@
       <artifactId>bigquery-connector</artifactId>
       <version>${bigquery.connector.hadoop2.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigtable</groupId>
+      <artifactId>bigtable-hbase-1.x-mapreduce</artifactId>
+      <version>${google.cloud.bigtable.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hbase</groupId>
+          <artifactId>hbase-server</artifactId>
```

Comment:
It's unfortunate that this requires server code, it seems like it should just be client logic. Do you know how much bigger this makes the jar?

Out of curiosity, what fails if we just exclude this without adding the shaded server dependency?

Reply:
Unfortunately, TableInputFormat and TableOutputFormat are both parts of server code